### ISSUE TITLE
Param used twice is not working

### ISSUE THREAD

**Issue by tcs-apeters** [Nachbarshund/node-mssql-connector#7]
select * from test where dscid IN (@dscid) or dscid IN (@dscid)

query.param("dscid", "Int", [5,6])

**Comment by nsharrok** [comment]
I get this error :+1:
```
q1.execute {"name":"param-not-found","message":"Param 'username' was not found in query or is tried to set twice"}
```
with this code
```javascript
    var conn = require('node-mssql-connector');
    var TYPES = require('tedious').TYPES
    sql = new conn( {...});
...
    var query = sql.query(
        "SELECT  staffID, commonName, firstName, surname, email " +
        "FROM staff_member " +
        "where  Username = @username ");
    query.param( "username", TYPES.NVarChar,  process.env.USERNAME );
    query.exec( function( err, rs ){
        if (err) return console.log("query.execute "+JSON.stringify(err));
        var staffID = rs[0].staffID;
...
```
if I comment out the param line I get
```
q1.execute {"name":"request-error","message":{"name":"RequestError","message":"Must declare the scalar variable \"@username\".","code":"EREQUEST","number":137,"state":2,"class":15,"serverName":"HRSHSQ
L01","procName":"","lineNumber":1}}
```
Loos like a bug to me
Noel

**Comment by Nachbarshund** [comment]
This only happened by using the IN statement. E.g. using parameter with the same name twice (without IN), it works correct:

```sql
SELECT *
FROM Tablename
WHERE ID=@id OR ID=@Id
```

This will be fixed in `v0.4.0`.
